instrumentation/netty/netty-4.1/src/main/java/io/opentelemetry/javaagent/instrumentation/hypertrace/netty/v4_1/NettyChannelPipelineInstrumentation.java

```diff
@@ -146,8 +146,8 @@ public static void addHandler(
           // add OTEL request handler to start spans
           pipeline.addAfter(
               HttpClientTracingHandler.class.getName(),
-              io.opentelemetry.javaagent.instrumentation.netty.v4_1.client
-                  .HttpClientRequestTracingHandler.class
+              io.opentelemetry.javaagent.instrumentation.netty.v4_1.client.HttpClientTracingHandler
```

Comment:
this was really hard to catch - this was basically always a copy-paste error but before now it didn't cause any trouble. On Spring webflux the `HttpClientRequestTracingHandler` has not been added at this point, but we should always have been adding this after the `HttpClientTracingHandler` anyways

Reply:
It caused the test to hang and would result in really odd hard to debug bahvior.